extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -772,7 +773,11 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
 
       const displaySet = displaySetService.getDisplaySetByUID(displaySetInstanceUID);
       if (!volume && displaySet.images) {
-        volume = csToolsUtils.getOrCreateImageVolume(displaySet.images.map(image => image.imageId));
+        const volumeLoaderSchema = displaySet.volumeLoaderSchema ?? VOLUME_LOADER_SCHEME;
+        const volumeId = `${volumeLoaderSchema}:${displaySet.displaySetInstanceUID}`;
```

Comment:
we don't need volume loader scheme anymore, you can try removing the left part of the `:` i think it should work